prody/proteins/header.py

```diff
@@ -915,7 +915,7 @@ def isSheet(secstrs):
     torf = secstrs == 'E'
     return torf
 
-def assignSecstr(header, atoms, coil=False):
+def assignSecstr(header, atoms, coil=False, report=True):
```

Comment:
If a user or a function wants to turn off the outputs, he/she/it can use `confProDy(debug='warning')` or `confProDy(debug='none')` to turn off the message and turn it back on after this code is run. Let's keep and stick to this ProDy style, otherwise, we should provide this argument for every function that prints out messages! (which is heavy work and unnecessary =) )

Reply:
I guess. I was thinking more in terms of what happens when we use higher level functions. Then we want the reporting of the lower level function to be turned off because we have other summary-level reporting instead. Generally reporting would be true to the extent set by confProDy when using the function by itself.

We can talk about this more but I feel we just need to add it to a function when we make another function that runs it multiple times.